mono/mini/debugger-agent.c

```diff
@@ -2734,7 +2727,7 @@ suspend_vm (void)
 
 	if (suspend_count == 1) {
 		// FIXME: Is it safe to call this inside the lock ?
-		start_single_stepping ();
+		mono_de_start_single_stepping ();
```

Comment:
mono_germany_start_single_stepping?
Can we use dbg rather than "de"? We don't use "de" to mean the debugger anywhere else.

Reply:
Or even debug or debugger or dbgeng. I also don't like "de", but I didn't speak up.
Some projects to have usually two character prefixes (Ex, Ps, Ke, Mm, Io) but it seemed obscure to me..but I realize what it is..probably we all do..

Reply:
I find "engine" a dubious part of any name. What do you call the non-engine parts?
Thing and ThingEngine? Thing and ThingCore? ThingFoo and ThingBar, but Foo and Bar are hard to name, so Foo is empty and Bar is Engine or Core?
.NET Engine?
d and de?
Dbg and DbgEng?